addons/payment_stripe/models/payment_transaction.py

```diff
@@ -143,7 +143,7 @@ def _stripe_prepare_payment_intent_payload(self):
             'currency': self.currency_id.name.lower(),
             'description': self.reference,
             'capture_method': 'manual' if self.provider_id.capture_manually else 'automatic',
-            'automatic_payment_methods[enabled]': True,
+            'expand[]': 'payment_method',
```

Comment:
why?

Reply:
to get payment method in expanded format to retrieve the payment method type in line 360

Reply:
I thought it was done in the controller :thinking:  It's great if it works like this ^^

Reply:
It needs to be done here too when paying by token.